packages/ui/components/pages/Fuse/Modals/Terms.tsx

```diff
@@ -0,0 +1,168 @@
+import {
+    Button,
+    UnorderedList,
+    ListItem,
+    List,
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+    Text,
+  } from '@chakra-ui/react'
+
+  import React from 'react';
+  import { useState, useEffect } from 'react';
```

Comment:
Fix prettier and eslint errors and warnings.

See suggested plugins in comment on pr.

`yarn workspace @midas-capital/ui lint`